examples/trpo_bullet_kuka.py

```diff
@@ -0,0 +1,57 @@
+from rllab.algos.trpo import TRPO
+from rllab.baselines.linear_feature_baseline import LinearFeatureBaseline
+from rllab.envs.bullet_env import BulletEnv
+from rllab.envs.normalized_env import normalize
+from rllab.misc.instrument import run_experiment_lite
+from rllab.policies.gaussian_mlp_policy import GaussianMLPPolicy
+
+from rllab.misc.rllab_gym_register import Register
+
+
+def run_task(*_):
+    # Please note that different environments with different action spaces may require different
```

Comment:
You can remove stale comments. Also I know that this is violated all over the codebase, but please don't exceed 80 character lines, even for comments.